lib/widget/dropdown.dart

```diff
@@ -227,9 +236,122 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
         focusNode: focusNode,
         decoration: inputDecoration);
     }
-    return const Text("Unimplemented SelectOne");
+    return LayoutBuilder(builder: (context, constraints) {
+      return RawAutocomplete<SelectOneItem>(
+        optionsBuilder: (TextEditingValue textEditingValue) {
+          return buildAutoCompleteOptions(textEditingValue);
+        },
+        displayStringForOption: (SelectOneItem option) =>
+        option.label ?? option.value,
+        fieldViewBuilder: (BuildContext context,
+            TextEditingController fieldTextEditingController,
+            FocusNode fieldFocusNode,
+            VoidCallback onFieldSubmitted) {
+          return TextField(
+              enabled: isEnabled(),
+              showCursor: true,
+              cursorColor:
+              widget._controller.cursorColor ?? EnsembleTheme.primary,
```

Comment:
This is the wrong way to getting the primary color. Instead use this `Theme.of(context).colorScheme.primary`. The color scheme can be changed at runtime which will update the entire app.